docs/streams/developer-guide.html

```diff
@@ -1573,7 +1576,7 @@
                             return aggValue + newValue;
                           }
                         },
-                        "reduced-stream-store" /* state store name */);
+                    );
```

Comment:
You must remove the comma `,` in the line before, now that we removed the last parameter.